src/components/menu/Menu.tsx

```diff
@@ -145,7 +153,7 @@ const Menu: GenericComponent<MenuProps> = ({
   const getMenuStyleByPosition = () => {
     switch (statePosition) {
       case POSITION.TOP_RIGHT:
-        return { clip: `rect(0 ${stateWidth}px 0 ${stateWidth}px)` };
+        return { clip: `rect(0 165.75px 0 165.75px)` };
```

Comment:
I assume this should be reverted?

Reply:
Yes it should, happened by mistake :D. Still working on it..